steps/identity/appellant-ibca-ref/content.cy.json

```diff
@@ -1,7 +1,7 @@
 {
     "titleHead": "Enter IBCA Reference number",
     "title": "Enter your IBCA Reference number",
-    "subtitle": "You can find your IBCA Reference number on your IRN letter.",
+    "subtitle": "You can find your IBCA Reference number on your Review Decision Notice.",
```

Comment:
should be "Review Decision Notice letter"

Reply:
Sahamat said I should drop 'letter' when I checked with him when starting